src/components/Hero.jsx

```diff
@@ -0,0 +1,122 @@
+/* eslint-disable react/prefer-stateless-function */
+/* eslint-disable space-before-blocks */
+/* eslint-disable jsx-a11y/anchor-is-valid */
+/* eslint-disable no-unused-expressions */
+/* eslint-disable react/jsx-indent */
+/* eslint-disable linebreak-style */
+import React, { Component } from 'react';
+import styled from 'styled-components';
+import { COLOR_MAIN_DARK, LAYOUT, COLOR_MAIN_LIGHT } from '../constants';
+import Button from './Button';
+import webpic from './ray_dalio_hero_web.jpg';
+import educatedPic from './educated_web_hero.jpg';
+import playstation from './playstation.jpg';
+import blackberry from './blackberry.jpg';
+import redbull from './redbull.jpg';
+import tiktok from './tiktok.jpg';
+import sennheiser from './sennheiser.jpg';
+
+
+
+const HeroWrapper = styled.div`
+    width: 100%;
+    background-color: ${COLOR_MAIN_DARK};
+
+`;
+
+const HeroContent = styled.div`
+    width: auto;
+    margin-left: auto;
+    margin-right: auto;
+    max-width: ${LAYOUT.desktop.maxWidth}px;
+    padding: ${LAYOUT.desktop.spacing}px;
+    overflow: hidden;
+
+    justify-content: space-between;
+    a {
+        text-decoration: none;
+        color: #fff;
+    }
+    
+`;
+
+const HeroLeft = styled.div`
+    font-size: 2rem;
+    font-weight: 350;
+    color: #fff;
+    width:350px;
+    float: left;
+    margin-left: 20px;
+    margin-top: 100px;
+`;
+
+const HeroRight = styled.div`
+    margin-right: 100px;
+    float: right;
+    display: inline-block;
+    margin-bottom: -800px;
+    img{
```

Comment:
It's quite subjective but by leaving an empty space between the CSS property and new target (margin-bottom and img), you'll make the code easier to read.

Also, you can add space after img and {, as there's a space in other places like that.
Consistency makes code beautiful :)